apps/web-giddh/src/app/shared/header/components/share-group-modal/share-group-modal.component.ts

```
@@ -57,11 +57,12 @@ export class ShareGroupModalComponent implements OnInit, OnDestroy {
         this.activeCompany$.pipe(takeUntil(this.destroyed$)).subscribe(activeCompany => {
             if (activeCompany) {
                 this.activeCompany = activeCompany;
-                if (activeCompany?.moduleRestrictionStatus) {
+                if (activeCompany?.subscription.planDetails.restrictedModules.hasOwnProperty(this.restrictedModules.Users) && activeCompany.moduleRestrictionStatus) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)